package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "gauge-js",
-  "version": "4.0.1",
+  "version": "5.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=18"
+  },
```

Comment:
I added this - is it a good idea, or will it lead to worse errors on installation? I guess we have the `install.js` plugin install, but we also have the direct dependency in people's projects, so this one would help address the addition to their projects.

Reply:
Ahh sorry, my bad - getting confused. Thanks for correcting me :-)